src/server/internalServer.h

```diff
@@ -148,7 +148,9 @@ class InternalServer {
 
     bool etag_not_needed(const RequestContext& r) const;
     ETag get_matching_if_none_match_etag(const RequestContext& request) const;
-    Library::BookIdSet selectBooks(const RequestContext& r, const std::string& prefix) const;
+    Library::
+
+    BookIdSet selectBooks(const RequestContext& r, const std::string& prefix) const;
```

Comment:
I guess that this is an accidental change

Reply:
Indeed